### PR TITLE
[17.0] [MIG] account_payment_mode

### PR DESCRIPTION
**Pull request by miquelalzanillas**

### CODE REVIEW

**Comment by miquelalzanillas** [comment]
cc https://github.com/APSL 148666

Please review: @peluko00 @lbarry-apsl @samoros1

**Review by samoros1 — APPROVED** [review]

**Review by lbarry-apsl — APPROVED** [review]
Validated functionality.

**Review by peluko00 — APPROVED** [review]

**Comment by peluko00** [comment]
Working good!

File: account_payment_mode/views/account_payment_method.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        <field name="name">account_payment_method.form</field>
+        <field name="model">account.payment.method</field>
+        <field name="arch" type="xml">
+            <form string="Payment Method">
+                <sheet>
+                    <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        invisible="active == True"
```
Comment (pedrobaeza) [inline account_payment_mode/views/account_payment_method.xml]: And the same for the other occurrences:

```suggestion
                        invisible="active"
```

Comment (miquelalzanillas) [reply account_payment_mode/views/account_payment_method.xml]: Thanks @pedrobaeza. Fixed.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**